### PR TITLE
fix bash syntax in olm catalog deploy script

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: olm_deploy/scripts/catalog-deploy.sh [hunk 1]
```diff
@@ -22,6 +22,8 @@ echo "elastic6: ${IMAGE_ELASTICSEARCH6}"
 echo "kibana: ${IMAGE_LOGGING_KIBANA6}"
 echo "oauth proxy: ${IMAGE_OAUTH_PROXY}"
 
+exit 0
```
Comment (ewolinetz) [inline olm_deploy/scripts/catalog-deploy.sh]: this `exit` seems preemptive...

Comment (bparees) [reply olm_deploy/scripts/catalog-deploy.sh]: oof. thank you, yes that was debug :(

**Pull request merged**